build.sbt

```diff
@@ -43,6 +43,7 @@ lazy val root = (project in file("."))
       "software.amazon.awssdk" % "lambda" % awsVersionTwo,
       "software.amazon.awssdk" % "auth" % awsVersionTwo,
       "software.amazon.awssdk" % "sts" % awsVersionTwo,
+      "software.amazon.awssdk" % "ec2" % awsVersionTwo,
```

Comment:
Note that this is likely to cause a conflict with other EC2 based collectors so you'll likely have to carry out some rebases and deal with the conflicts.